onnxruntime/core/common/cpuid_info.cc

```diff
@@ -188,6 +188,7 @@ void CPUIDInfo::ArmLinuxInit() {
     has_arm_neon_dot_ = cpuinfo_has_arm_neon_dot();
     has_fp16_ = cpuinfo_has_arm_neon_fp16_arith();
     has_arm_neon_i8mm_ = cpuinfo_has_arm_i8mm();
+    has_arm_sve_ = cpuinfo_has_arm_sve();
```

Comment:
I don't see this in `ArmWindowsInit()` and `ArmAppleInit()` -> is it intentional to limit these to just Linux ?